### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by vechicin**
This is my first capstone project during my Microverse career.

# Features

- There's two pages: a Home page and an About page
- They have been created to be responsive for two screen widths: a mobile version (up to 768px) and a desktop version (starting at 768px)
- The mobile version has an interactive mobile menu
- The desktop version's featured speakers section was built using dynamic HTML

# Video Presentation
Here's the [link](https://www.loom.com/share/29b6a96347e94dc585981d9015b7116a) to the video presentation for the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Capstone
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is the Capstone Project for my first module in Microverse. This is a page about a fictional jazz concert where previously unreleased footage will be debuted for the world to see.
+
+![screenshot](./assets/images/screenshot.png)
+
+If you want to check the full page, please follow [this link](https://vechicin.github.io/Capstone/). 
+
```
Comment (rotimiazeez) [inline README.md]: - [OPTIONAL] Please be sure to add the link of the video to your README file😉

File: about.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  <section class="partnerships">
+    <h2 class="partnerships-title">In Partnership With:</h2>
+    <div class="partner-logos">
+      <img class="downbeat" src="./assets/images/4570723_orig.png" alt="Downbeat Magazine Logo">
+      <img class="montreux"
+        src="./assets/images/52nd-montreux-jazz-festival-switzerland-shutterstock-editorial-9735005bf.jpeg"
+        alt="Montreux Jazz Festival Logo">
+    </div>
+  </section>
+</body>
```
Comment (rotimiazeez) [inline about.html]: - [ ] Please note that both pages contain the same navbar and footer✌.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      <p class="email-address">jazzmasters@jazzmasters.com</p>
+    </article>
+  </main>
+
+  <section class="logo">
+    <h2 class="logo-title">JazzMasters Logo</h2>
+    <p class="logo-description">The logo for the first edition of the JazzMasters concert was decided through the
+      international JazzMasters Logo competition.</p>
+    <img class="logo-image" src="./assets/images/istockphoto-607972154-612x612.jpeg" alt="The JazzMasters logo">
+  </section>
```
Comment (rotimiazeez) [inline about.html:95]: - [ ] Please note that the about page should contain a bottom section with a centred column that includes a section header, a divider, a line of text, and two images with text centred over them😉.
Like this👇🏽
behance

Comment (rotimiazeez) [reply about.html:95]: Please note that on mobile the bottom section images are organized in a vertical column👌 and on desktop, the bottom section images are organized in a horizontal row😉. [See design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

File: about.css [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+
+  .email-address {
+    display: flex;
+    justify-content: center;
+  }
+
+  .logo-description {
+    text-align: center;
+  }
+}
```
Comment (rotimiazeez) [inline about.css]: - [ ] Please make sure the CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around.🙏🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @vechicin 👋, 

### Good job so far!👏👏


### SUMMARY❤
- Good README file👏
- No linter error ✔
- Good PR description👍
- Great UI✌😎

### There are some issues that you still need to work on to go to the next project but you are almost there!💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.css [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+
+  .desktop-title {
+    display: none;
+  }
+
+  .navbar-items {
+    display: none;
+  }
+}
+
```
Comment (AshakaE) [inline about.css]: Building a mobile first app means writing your media queries from bottom up i.e using `min-width` , everything above the `min-width` will make for a larger screen with specific breakpoints and vice versa.

- [ ] Please use the mobile-first approach only 🙂

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @vechicin  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — APPROVED** [review]
Hi @vechicin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**